README.md

```diff
@@ -166,7 +178,7 @@ snackProgressBarManager.put(snackProgressBar, storeId)
 snackProgressBarManager.show(storeId, SnackProgressBarManager.LENGTH_LONG)
 ```
 
-Calling `show()` will put the SnackProgressBar into a queue, which will be shown after those in queue before it has been dismissed
+Calling `show()` will put the SnackProgressBar into a queue, which will be shown after those in queue before it have been dismissed
```

Comment:
Calling `show()` will put the SnackProgressBar into a queue, which will be shown after other queued SnackProgressBars.